### ISSUE TITLE
Change attribute ldif to better support inetOrgPerson

### ISSUE THREAD

**Issue by nynymike** [GluuFederation/community-edition-setup#282]
In metadata for `inetOrgPerson` attributes which are not included in `gluuPerson`, change `gluuAttributeOrigin` to `gluuCustomPerson`

Customers who want to add these attributes should just add them to the gluuPerson LDAP schema.

```
gluuAttributeName: carLicense
gluuAttributeName: facsimileTelephoneNumber
gluuAttributeName: departmentNumber
gluuAttributeName: employeeType
gluuAttributeName: cn
gluuAttributeName: mobile
gluuAttributeName: telephoneNumber
gluuAttributeName: st
gluuAttributeName: manager
gluuAttributeName: street
gluuAttributeName: postOfficeBox
gluuAttributeName: employeeNumber
gluuAttributeName: preferredDeliveryMethod
gluuAttributeName: telephoneNumber
gluuAttributeName: roomNumber
gluuAttributeName: secretary
gluuAttributeName: homePostalAddress
gluuAttributeName: l
gluuAttributeName: postalCode
gluuAttributeName: description
gluuAttributeName: title
```

**Comment by tecoholic** [comment]
@yurem Can you kindly take a look at this? I think we need update https://github.com/GluuFederation/community-edition-setup/blob/master/schema/gluu_schema.json

**Comment by yurem** [comment]
@tecoholic Mike offered to update 'gluuAttributeOrigin' in https://github.com/GluuFederation/community-edition-setup/blob/master/templates/attributes.ldif

**Comment by aliaksander-samuseu** [comment]
@yurem @shekhar16 

Please note that I also can reproduce the issue in the latest 3.1.0 deb beta package, we should not forget to apply this fix there to.

**Comment by aliaksander-samuseu** [comment]
@yurem @shekhar16 @sahiliamsso 

I can confirm that at least "employeeNumber" attribute's metadata with incorrect objectclass ("inetOrgPerson") in it is still present in LDAP in 3.0.2 deb package causing issue, probably others' too. Did we test it properly?

**Comment by tecoholic** [comment]
Here is the diff that shows that it has been changed in `attributes.ldif`

![closes__282__updated](https://user-images.githubusercontent.com/383862/27803849-3bf86d26-6049-11e7-9cee-eed6ca47a989.png)


Is there any other place where this change needs to be reflected?

**Comment by aliaksander-samuseu** [comment]
@tecoholic 

Just a few results from search like this `# /opt/opendj/bin/ldapsearch -h 127.0.0.1 -p 1636 -s sub -T -Z -X -D 'cn=directory manager,o=gluu' -j '/tmp/.pw' -b 'ou=attributes,o=@!80E9.CD48.2755.A503!0001!A098.8C39,o=gluu' -z 5 '&(objectclass=gluuattribute)(gluuAttributeOrigin=inetOrgPerson)' gluuAttributeOrigin gluuAttributeName` against my 3.0.2 Ubuntu 14.04 instance:

```
dn: inum=@!80E9.CD48.2755.A503!0001!A098.8C39!0005!C001,ou=attributes,o=@!80E9.CD48.2755.A503!0001!A098.8C39,o=gluu
gluuAttributeName: title
gluuAttributeOrigin: inetOrgPerson

dn: inum=@!80E9.CD48.2755.A503!0001!A098.8C39!0005!4502,ou=attributes,o=@!80E9.CD48.2755.A503!0001!A098.8C39,o=gluu
gluuAttributeName: carLicense
gluuAttributeOrigin: inetOrgPerson

dn: inum=@!80E9.CD48.2755.A503!0001!A098.8C39!0005!BD22,ou=attributes,o=@!80E9.CD48.2755.A503!0001!A098.8C39,o=gluu
gluuAttributeName: facsimileTelephoneNumber
gluuAttributeOrigin: inetOrgPerson

dn: inum=@!80E9.CD48.2755.A503!0001!A098.8C39!0005!08E2,ou=attributes,o=@!80E9.CD48.2755.A503!0001!A098.8C39,o=gluu
gluuAttributeName: departmentNumber
gluuAttributeOrigin: inetOrgPerson

dn: inum=@!80E9.CD48.2755.A503!0001!A098.8C39!0005!3D44,ou=attributes,o=@!80E9.CD48.2755.A503!0001!A098.8C39,o=gluu
gluuAttributeName: employeeType
gluuAttributeOrigin: inetOrgPerson
```

**Comment by aliaksander-samuseu** [comment]
@tecoholic 

I see it's supposedly fixed for "master" branch, which perhaps means for 3.1.x. Was it also back-ported to 3.0.2? As milestone for this report set to "3.0.2" I tested it in 3.0.2 as well.

**Comment by tecoholic** [comment]
@aliaksander-samuseu OK. Got it. I think the issue is with the fix not being present in the 3.0.2 branch. I will back port it to the 3.0.x branch when we have one.

**Comment by tecoholic** [comment]
For now I have pushed the commit to both `version_3.0.2` and `version_3.0.2.1` branches. So if any new package is built or any new branch is created from any one of these two or the master in the future, the fix would be present in it.
